apps/website/src/pages/features/wallet.tsx

```diff
@@ -0,0 +1,95 @@
+            image="/images/wallet/5.png"
+            imageAlt="wallet-5"
+            imageSecondary="/images/wallet/pizza.png"
+            imageSecondaryAlt="pizza"
+            secondaryDescription="Status automatically calculates the cheapest way of moving tokens from A to B, considering: gas prices, chains your tokens are on, chains the recipient uses, and bridge costs. All in real time."
+            secondaryTitle="Cost efficient"
+            direction="rtl"
+          />
+        </div>
+        <div className="relative py-[160px]">
+          <TwoColsSection
+            title="Full control for power users"
+            description="Select exact amounts to send from each blockchain, and precisely set the chains the recipient will receive funds on."
+            image="/images/wallet/6.png"
+            imageAlt="wallet-6"
+            imageSecondary="/images/wallet/duck.png"
+            imageSecondaryAlt="duck"
+            secondaryDescription="Gas limit, max base fee, max priority fee, input data and nounce value for every transaction in each transaction bundle. If you really want to..."
+            secondaryTitle="Pro user? You can edit:"
+          />
+          <ParalaxCircle initialRight={-200} initialTop={-300} />
+        </div>
+
+        <ComparisionSection />
+        <div className="border-neutral-10 bg-white-100 relative rounded-[48px] border-t py-[160px] mix-blend-normal shadow-[0_-2px_20px_rgba(9,16,28,0.04)]">
+          <TwoColsSection
+            title="New address format"
+            description="A way for wallets to automatically signal which chains they can receive funds on. So you don’t have to."
+            image="/images/wallet/7.png"
+            imageAlt="wallet-7"
+            imageSecondary="/images/wallet/cube.png"
+            imageSecondaryAlt="cube"
+            secondaryDescription="Status is working with other wallets to ensure EIP-3770 with multiple chain shortName prefixes becomes the new ethereum address standard."
+            secondaryTitle="On the path to adoption"
+            direction="rtl"
+          />
+          <div className="relative pt-[160px]">
+            <TwoColsSection
+              title="Total balance graphs"
+              description="Who doesn’t want to see how their ‘crypto portfolio number’ has gone up or down over time?"
+              image="/images/wallet/8.png"
+              imageAlt="wallet-8"
+              imageSecondary="/images/wallet/megaphone.png"
+              imageSecondaryAlt="megaphone"
+              secondaryDescription="Everything is taken into account: fluctuations in fiat exchange rates, daily valuation updates of every crypto asset you hodl, and how your balance changes when tokens are sent/received"
+              secondaryTitle="Graphs that do maths"
+            />
+          </div>
+        </div>
+        <HandsSection />
+      </div>
+    </>
```

Comment:
No need for Fragment `<>`

Reply:
It was, but the markup changed and i forgot to fix that.

Reply:
Yeah, I do that all the time